minoc/src/main/java/org/opencadc/minoc/ArtifactAction.java

```diff
@@ -122,7 +126,9 @@
     static final String SCHEMA_KEY = SQLGenerator.class.getPackage().getName() + ".schema";
     static final String READ_GRANTS_KEY = ReadGrant.class.getName() +  ".resourceID";
     static final String WRITE_GRANTS_KEY = WriteGrant.class.getName() +  ".resourceID";
-    
+    private static final URI GMS_SERVICE_URI =
```

Comment:
single line length is 120 in checkstyle, isn't it? IDE config?

Reply:
I'd prefer no line length limit myself, I thought we agreed on 120, but my IDE wants 80. My check style plugin may have lost our checkstyle config file after an upgrade. Will fix my IDE and reformat my changes to 120.